### PR TITLE
Update view on activation with block at current chain height

### LINKED ISSUE #2408: Attempting to make a proposal incorrectly indicated to wait until next proposal phase
I went to submit a proposal and the `Next proposal phase` was empty:
image

Looking at the voting cycle overview, I should have been able to submit a proposal as the block height was still within the proposal phase:
image

### PR DESCRIPTION
**Pull request by ripcurlx**
Fixes #2408.

### CODE REVIEW

File: core/src/main/java/bisq/core/dao/DaoFacade.java [hunk 1]
```diff
@@ -484,6 +485,15 @@ public int getChainHeight() {
         return daoStateService.getChainHeight();
     }
 
+    public Block getBlockAtChainHeight() {
+        return getBlockAtHeight(getChainHeight());
+    }
+
+    public Block getBlockAtHeight(int chainHeight) {
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/DaoFacade.java]: Please add @Nullable

Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/dao/DaoFacade.java]: Or stick with Optional...

**Review by ManfredKarrer — APPROVED** [review]
Beside the @Nullable/Optional issue utACK

File: core/src/main/java/bisq/core/dao/DaoFacade.java (hunk 1 above)
Comment (ripcurlx) [reply core/src/main/java/bisq/core/dao/DaoFacade.java]: I've already changed everything to Optional. Please see my force pushed state.

**Review by ManfredKarrer — APPROVED** [review]
Ah saw you pushed that already....

**Pull request merged**